src/ImageSharp/Quantizers/Wu/WuQuantizer.cs

```diff
@@ -773,7 +773,7 @@ private QuantizedImage<TColor> GenerateResult(PixelAccessor<TColor> imagePixels,
             Parallel.For(
                 0,
                 height,
-                Bootstrapper.ParallelOptions,
+                Bootstrapper.Default.ParallelOptions,
```

Comment:
looks like all the `Quantizer`s will probably need tweeking to pass along the bootstrapper, the ` IQuantizer<TColor>.Quantize(..)` method looks like the best place to me, and then proxied on to the `QuantizedImage` constructor.